### PR TITLE
change border color

### PR DESCRIPTION
**Pull request by LluisBagur**

### CODE REVIEW

**Comment by seand52** [comment]
nop

File: index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
             color: white;
             background-color: orange;
-            border: 10px solid skyblue;
+            border: 10px solid greenyellow;
```
Comment (seand52) [inline index.html]: no me gusta este color

File: index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
             color: white;
```
Comment (seand52) [inline index.html]: hey move styles to file!

Comment (LluisBagur) [reply index.html]: true, but both should have done thet, how we proced then?

Comment (seand52) [reply index.html]: no problem. i proceed to move styles to file, after merge

**Pull request merged**